From 48e218d9340ff88342a3f3e2371785b8cd527d1f Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:28:54 -0400
Subject: [alsa] asihpi: fix some indenting in snd_card_asihpi_pcm_new()

Message-id: <140972823573-246-patchset-jkysela@redhat.com>
Patchwork-id: 90946
O-Subject: [RHEL 7.1 PATCH 245/442] ALSA: asihpi: fix some indenting in snd_card_asihpi_pcm_new()
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

This used to be a part of a condition until f3d145aac913 ('ALSA: asihpi:
MMAP for non-busmaster cards') but now it's not and we can remove an
indent level.

Signed-off-by: Dan Carpenter <dan.carpenter@oracle.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: c687c9bbda373bfab8c8df8362a27b1ecb5240c1)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/asihpi/asihpi.c b/sound/pci/asihpi/asihpi.c
index e4e42f2..901c949 100644
--- a/sound/pci/asihpi/asihpi.c
+++ b/sound/pci/asihpi/asihpi.c
@@ -1253,11 +1253,12 @@ static int snd_card_asihpi_pcm_new(struct snd_card_asihpi *asihpi, int device)
    num_outstreams, num_instreams, &pcm);
  if (err < 0)
   return err;
+
  /* pointer to ops struct is stored, dont change ops afterwards! */
-  snd_pcm_set_ops(pcm, SNDRV_PCM_STREAM_PLAYBACK,
-    &snd_card_asihpi_playback_mmap_ops);
-  snd_pcm_set_ops(pcm, SNDRV_PCM_STREAM_CAPTURE,
-    &snd_card_asihpi_capture_mmap_ops);
+ snd_pcm_set_ops(pcm, SNDRV_PCM_STREAM_PLAYBACK,
+   &snd_card_asihpi_playback_mmap_ops);
+ snd_pcm_set_ops(pcm, SNDRV_PCM_STREAM_CAPTURE,
+   &snd_card_asihpi_capture_mmap_ops);
 
  pcm->private_data = asihpi;
  pcm->info_flags = 0;
-- 
1.7.1